components/pulp-access-controller/README.md

```diff
@@ -0,0 +1,3 @@
+# Pulp Access Controller
+
+The `pulp-access-controller` is a CRD/Controller which creates namespace secrets that provide access to Pulp over TLS.
```

Comment:
```suggestion
The `pulp-access-controller` is a CRD/Controller which creates namespace secrets that provide access to Pulp over TLS.

It is similar in concept to `image-controller`. Image Controller provisions quay repos and creates secrets for user pipelines to use. Pulp Access Controller provisions pulp tenants and provides secrets for user pipelines to access those.
```

Reply:
My hope is that this sentence helps Konflux contributors understand the role of this thing.